src/bindings/python/BUILDING.md

```diff
@@ -1,88 +1,145 @@
-# Building the nGraph Python* API
+# Building the OpenVINO™ Python* API
 
-This document provides the instructions for building the nGraph Python API from source on Linux, macOS and Windows 10 platforms.
+This document provides the instructions for building the OpenVINO™ Python API from source on Linux, macOS and Windows 10 platforms.
 
 For each platform, you can build and install the API as a part of OpenVINO™ Toolkit or as a Python wheel.
-A Python wheel is a portable package that allows you to install nGraph in your Python distribution, or dedicated virtual environment.
+A Python wheel is a portable package that allows you to install OpenVINO™ in your Python distribution, or dedicated virtual environment.
 
 ## Linux* and macOS*
 
 ### Prerequisites
 
-To build the nGraph Python API, you need to install a few additional packages.
+To build the OpenVINO™ Python API, you need to install a few additional packages.
 
 On Ubuntu* 20.04 LTS you can use the following instructions to install the required packages, including Python and Cython.
 
-    apt install git wget build-essential cmake
-    apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
+```bash
+apt install git wget build-essential cmake
+apt install python3 python3-dev python3-pip python3-virtualenv python-is-python3
```

Comment:
why not to use `openvino_build_dependencies.sh`?

Reply:
It is mentioned later on, if carefully read.